scripts/build_demand_profiles.py

```diff
@@ -315,6 +391,9 @@ def upsample(cntry, group):
     start_date = snakemake.params.snapshots["start"]
     end_date = snakemake.params.snapshots["end"]
     out_path = snakemake.output[0]
+    # fulmicotone
+    file_path_temba = r"C:\Users\Davide\Downloads\temba\TEMBA_SSP1-26.xlsx"
+    # year = 2030
```

Comment:
To clarify the above comment on Snakemake inputs: it's currently not obvious where `file_path_temba` is defined, and super easy to forget modify it if anything has been changed. Having this specification in Snakemake would facilitate keeping everything in sight